README.md

```diff
@@ -54,6 +54,8 @@ The org structure is represented in a series of columns `org1..orgN` where `N` i
 
 There is repetition in organizational hierarchy in this format. The reference implementation validates the internal consistency of those organizational repositories across CSV lines.
 
+You can either overwrite the existing [repos.csv](src/main/resources/repos.csv) or set the `moderne.scm.reposCsvPath` in [application.yaml](src/main/resources/application.yaml).
```

Comment:
Does it make sense for it to be in the `scm` namespace?

Reply:
hmm, no actually not 🤔

Reply:
I will extract it